lua/auto-dark-mode/init.lua

```diff
@@ -71,6 +73,9 @@ end
 
 local function init()
 	system = vim.loop.os_uname().sysname
+	if string.match(vim.loop.os_uname().release, 'WSL') then
+		system = "WSL"
+	end
```

Comment:
It would be better to do an early return here to avoid executing `vim.loop.os_uname().sysname` in case it's not required :) 

```suggestion
	if string.match(vim.loop.os_uname().release, 'WSL') then
		system = "WSL"
		return
	end
	system = vim.loop.os_uname().sysname
```

Reply:
Thank you for the suggestion. The logic for detecting the WSL environment has been simplified.